.auto-release.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+repo_slug="garyyu/grin"
```

Comment:
Shouldn't this be `mimblewimble/grin`?

Reply:
Ah, Yes!   this one is forgot 😄   Thanks @jaspervdm

Reply:
fixed now.